hdmf-dev/hdmf
# Add/mailmap

## Motivation

Some names and emails in the commit log are bad. This adds a `.mailmap` file such that we can fix the mapping of names and emails so they display correctly with gitlog

## How to test the behavior?
Run `git show 08360f8494ce97ffc965015499ddea052decdaef` which shows `!git for-eac....` as the author name

```
commit 08360f8494ce97ffc965015499ddea052decdaef
Author: !git for-each-ref --format='%(refname:short)' `git symbolic-ref HEAD` <ben.dichter.......
```

With the updated `.mailmap` the name is displayed correctly as:

```
commit 08360f8494ce97ffc965015499ddea052decdaef
Author: Ben Dichter <ben.dichter....
```

## Checklist

- [X] Have you checked our [Contributing](https://github.com/hdmf-dev/hdmf/blob/dev/docs/CONTRIBUTING.rst) document?
- [X] Is your contribution compliant with our coding style? This can be checked running `ruff` from the source directory.
- [X] Have you checked to ensure that there aren't other open [Pull Requests](https://github.com/hdmf-dev/hdmf/pulls) for the same change?
